frontend/packages/client/src/components/Community/CommunityEditorDetails.js

```diff
@@ -26,11 +26,7 @@ export const CommunityUsersForm = ({
 } = {}) => {
   const canDeleteAddress = addrList.length > 1;
   return (
-    <WrapperResponsive
-      classNames="border-light rounded-lg columns is-flex-direction-column is-mobile m-0"
-      extraClasses="p-6 mb-6"
-      extraClassesMobile="p-4 mb-4"
-    >
+    <div className="border-light rounded-lg columns is-flex-direction-column is-mobile m-0 p-6 mb-6 p-4-mobile mb-4-mobile">
```

Comment:
here we have ´p-4-mobile mb-4-mobile´ applied on mobile and for the rest p-6 and m-0 would be applied